client_configuration_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+
+package gosnowflake
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+func TestParseConfiguration(t *testing.T) {
+	dir := t.TempDir()
+	testCases := []struct {
+		Name             string
+		FileName         string
```

Comment:
Do we need this? Maybe overwrite mode is good enough and we can get rid of this parameter?

Reply:
we agreed to leave it as is